src/main/java/org/jasig/portlet/newsreader/mvc/AbstractNewsController.java

```diff
@@ -50,16 +101,18 @@ public boolean isAdmin(PortletRequest req) {
     @ModelAttribute("isGuest")
     public boolean isGuest(PortletRequest req) {
         return req.getRemoteUser() == null;
-    }    
+    }
 
     @ModelAttribute("supportsEdit")
     public boolean supportsEdit(PortletRequest req) {
-        return req.isPortletModeAllowed(PortletMode.EDIT);
-    }    
+        final PortletPreferences prefs = req.getPreferences();
+        final String allowEdit = prefs.getValue(ALLOW_EDIT_PREFERENCE, "true");
+        return Boolean.parseBoolean(allowEdit);
+    }
```

Comment:
This is OK but is technically an API change and should probably bump at least the minor version # in the pom.xml.